idea_town/frontend/static-src/app/lib/web-channel.js

```diff
@@ -25,7 +25,7 @@ class WebChannel {
 
     const message = evt.detail;
     if (message && message.type) {
-      app.trigger('webChannel:' + message.type, message.data);
+      app.trigger('webChannel:' + message.type, message.detail);
```

Comment:
might be a typo? message is created from `evt.detail` on line 26. Do our messages have the form evt.detail.detail?